openwisp_notifications/handlers.py

```diff
@@ -24,9 +25,16 @@ def notify_handler(verb, **kwargs):
     public = bool(kwargs.pop('public', True))
     description = kwargs.pop('description', None)
     timestamp = kwargs.pop('timestamp', timezone.now())
-    level = kwargs.pop('level', Notification.LEVELS.info)
     recipient = kwargs.pop('recipient', None)
 
+    notification_template = NOTIFICATION_TYPES.get(kwargs.pop('type', None), None)
```

Comment:
I think we need to add a field to the model as well.
Make sure it's indexed with `db_index=True`